### PR TITLE
[NCF] Add run_eagerly for ctl.

### PR DESCRIPTION
**Pull request by tfboyd**

### CODE REVIEW

File: official/recommendation/ncf_keras_benchmark.py [hunk 1]
```diff
@@ -147,6 +147,14 @@ def benchmark_1_gpu_ctl_early_stop(self):
     FLAGS.early_stopping = True
     self._run_and_report_benchmark()
 
+  def benchmark_1_gpu_ctl_default_ds_run_eagerly_early_stop(self):
+    self._setup()
+    FLAGS.distribution_strategy = 'default'
```
Comment (guptapriya) [inline official/recommendation/ncf_keras_benchmark.py]: nice! did the default ds work out of the box?

Comment (tfboyd) [reply official/recommendation/ncf_keras_benchmark.py]: yes and I just renamed the test.

**Comment by tfboyd** [comment]
Fixed and updated the comment on the test.  PTAL

**Comment by nnigania** [comment]
nice, this will allow us to run eager with CTL. Just has a question, I thought eager mode does not support distribution strategy?

File: official/recommendation/ncf_keras_benchmark.py [hunk 2]
```diff
@@ -181,6 +181,13 @@ def benchmark_1_gpu_ctl_early_stop(self):
     FLAGS.early_stopping = True
     self._run_and_report_benchmark()
 
+  def benchmark_1_gpu_ctl_run_eagerly_early_stop(self):
```
Comment (guptapriya) [inline official/recommendation/ncf_keras_benchmark.py]: Do we also want to do a non dist strat run eagerly?

Comment (tfboyd) [reply official/recommendation/ncf_keras_benchmark.py]: Now I am confused again; and now I remember dist strat needs turned off.  I always look at this branch last when I am rushing to finish other stuff.  One more try coming up.  :-)

Comment (tfboyd) [reply official/recommendation/ncf_keras_benchmark.py]: Other than removing tests this is the last test I am adding period... probably....no seriously...maybe.

**Comment by nnigania** [comment]
hi,

the changes look good to me. Should be go ahead and checkin this change?

**Pull request merged**